CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+- Drop Ruby 2.7 and Rails 5 support. ([@palkan][])
```

Comment:
Looks like changelog is meant to be ruby older than 2.7 rather than 2.7 itself. Based on other changes in PR